pkgs/games/augustus/default.nix

```diff
@@ -19,6 +19,7 @@ stdenv.mkDerivation rec {
     homepage = "https://github.com/Keriew/augustus";
     license = licenses.agpl3;
```

Comment:
`agpl3` is a deprecated license, check if project uses `agpl3Plus` or `agpl3Only` and change `meta.license` accordingly.

Reply:
Changed it to agpl3Only, as there is a copy of the AGPL v3 in the repo, and no mention of "or any later version" anywhere.